managed/src/main/resources/reference.conf

```diff
@@ -56,6 +56,7 @@ db {
 yb {
   mode="PLATFORM"
   use_kubectl=true
+  use_helm_new_naming=false
```

Comment:
I would also add this to the runtime config section below, which allows setting it without a restart. Setting it would change the behavior for new univs created from the non.